marlowe/doc/playground-overview.adoc

```diff
@@ -19,119 +19,172 @@ contract in the browser-based tool, the Marlowe Playground, a web tool
 
-Assuming that we do select *Apply* we then see at the foot of the screen that the deposit has taken place
+=== Simulating Marlowe contracts
+
+However a contract is written, when it is sent to simulation this is the view seen first.
 
-image:sim-3.png[Simulation step 3]
+image:simulation-tab.png[The Simulation pane]
 
-and this remains in view if we add Alice's and Bob's choices to the transaction. Note also the the current state of the contract is shown in the main part of the window, and indeed we are waiting at this stage for choices from Alice and Bob.
+The code shown here presents whatever of the contract remains to be simulated, and the pane at the foot gives data about the simulation, as well as giving access to _static analysis_ for the contract (from its current state).
 
-If Alice and Bob make different choices, and we apply the transaction we then see
+Simulation is started by clicking the *Start simulation* button, and once this is done, the available actions that will advance the contract are presented;
 
-image:sim-4.png[Simulation step 4]
+image:available-actions.png[The actions available, width=300]
+
+In this case there are two potential actions: Alice can make a deposit, or the slot (time) can advance to `10` at which the wait for a deposit times out. Two other generic actions can be taken too
+
+* *Undo* will undo the last action made in the simulator. This means that we can explore a contract interactively, making some moves, undoing some of them, and then proceeding in a different direction.
+* *Reset* will reset the contract and its state back to their initial values: the full contract and an empty state.
+
+For our example, let us select for Alice to make the deposit of 450 lovelace. We can do that with the *+* button next to this input. After doing that we see
+
+image:simulation2.png[Simulation step 2]
+
+Where we see at the foot of the screen that the deposit has taken place. 
+
+This remains in view if we then make Alice's and then Bob's choice. Note also the the current state of the contract is shown in the main part of the window, and indeed we are waiting at this stage for a choice from Alice.
+
+If Alice and Bob make different choices we then see
+
+image:simulation3.png[Simulation step 3]
 
 and at this point in the evolution of the contract we are awaiting a choice from Carol to arbitrate the outcome.
 
+=== Saving and Opening Projects
+
+Projects are saved on github, and so when you first save a project you will be prompted thus:
+
+image:github1.png[Prompt to login to github]
+
+and if you choose to *Login* there you will be taken to a login screen for github:
```

Comment:
and, if you choose to *Login* there, you ...

Reply:
done